### PR TITLE
Parallel guests install

### PR DESCRIPTION
**Pull request by tbaev**
The network issues seems to be resolve and tests run a lot more stable, reverting back to parallel install of guests to run tests faster. The root cause of the network problem is still unknown.

- Related ticket: https://progress.opensuse.org/issues/101256
- Needles: N/A
- Verification run: [XEN](http://openqa.qam.suse.cz/tests/31608#) and [KVM](http://openqa.qam.suse.cz/tests/31619)

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]
LGTM :rocket:

**Comment by grisu48** [comment]
Adding @tonyyuan1 as reviewer as well

File: tests/virtualization/universal/patch_guests.pm
Comment (pdostal) [inline tests/virtualization/universal/patch_guests.pm:37]: We already loop over `%virt_autotest::common::guests` on line 34.

**Review by pdostal — CHANGES_REQUESTED** [review]

File: tests/virtualization/universal/patch_guests.pm
Comment (tbaev) [reply tests/virtualization/universal/patch_guests.pm:37]: I will remove line 34 and rerun the VR, thanks.

**Comment by tonyyuan1** [comment]
LGTM :)

**Comment by alice-suse** [comment]
@tbaev Suggest to merge the three commits into one before merge.

**Review by pdostal — APPROVED** [review]
Now it looks perfect :+1:

**Comment by grisu48** [comment]
LGTM :rocket: Let's merge this after the current ongoing runs

**Review by pdostal — APPROVED** [review]

**Comment by alice-suse** [comment]
@tbaev Can you trigger typical jobs in openqa.qam.suse.cz to double check all is good with the code?

**Comment by tbaev** [comment]
@alice-suse VR runs with the changes [KVM](http://openqa.qam.suse.cz/tests/32563#) and [XEN](http://openqa.qam.suse.cz/tests/32552#)

**Pull request merged**
